app/src/main/java/at/roteskreuz/stopcorona/screens/dashboard/DashboardFragment.kt

```diff
@@ -231,16 +211,11 @@ class DashboardFragment : BaseFragment(R.layout.fragment_dashboard) {
         }
     }
 
-    // TODO: 27/05/2020 dusanjencik: Remove
-    private fun checkPlayServicesAvailabilityAndStartHandshakeFragment() {
+    private fun checkPlayServicesAvailabilityAndRegisterToExposureNotificationFramework(enableFramework: Boolean) {
```

Comment:
I believe this should be handled in another Ticket: https://tasks.pxp-x.com/browse/CTAA-1363 and https://tasks.pxp-x.com/browse/CTAA-1395

Reply:
Agree. In the code, there is already a todo comment for it: `// TODO: 28/05/2020 dusanjencik: We should check also correct version`